net/kea/Makefile

```diff
@@ -0,0 +1,201 @@
+#
+# Copyright (C) 2019 Banglang Huang <banglang.huang@foxmail.com>
+# Copyright (C) 2019 Rosy Song <rosysong@rosinson.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=kea
+PKG_VERSION:=1.5.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=BangLang Huang<banglang.huang@foxmail.com>, Rosy Song<rosysong@rosinson.com>
+PKG_BUILD_DEPENDS:=log4cplus kea/host
+HOST_BUILD_DEPENDS:=log4cplus/host boost/host
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://ftp.isc.org/isc/kea/$(PKG_VERSION)/
+PKG_HASH:=edce4fab68ca7af607cf7f5bc86596e04fe0ef4b8e88906e339cdefcf21daaec
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_LICENSE:=MPL-2.0
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+
+HOST_BUILD_PREFIX:=$(STAGING_DIR_HOST)
+
+define Package/kea/Default
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=IP Addresses and Names
+  TITLE:=ISC Kea
+  URL:=https://www.isc.org/kea
+endef
+
+define Package/kea/description
+  Kea is an open source DHCPv4/DHCPv6 server being developed by Internet
+  Systems Consortium. Kea is a high-performance, extensible DHCP server
+  engine that is designed to be easily modified and extended with hooks
+  libraries. It provides DHCPv4 and DHCPv6 servers, a dynamic DNS update
+  module, a portable DHCP library, libdhcp++, control agent that provides
+  management REST interface, and a DHCP benchmarking tool, perfdhcp.
+endef
+
+define Package/kea-libs
+	$(call Package/kea/Default)
+	TITLE+= Libraries
+	DEPENDS:=+libopenssl +log4cplus \
+		+boost +boost-python3 +boost-system
+endef
+
+define Package/kea-dhcp4
+	$(call Package/kea/Default)
+	TITLE+= DHCP Server v4
+	DEPENDS:=+kea-libs
+endef
+
+define Package/kea-dhcp6
+	$(call Package/kea/Default)
+	TITLE+= DHCP Server v6
+	DEPENDS:=@IPV6 +kea-libs
+endef
+
+define Package/kea-dhcp-ddns
+	$(call Package/kea/Default)
+	TITLE+= DHCP - DDNS
+	DEPENDS:=+kea-libs
+endef
+
+define Package/kea-admin
+	$(call Package/kea/Default)
+	TITLE+= Admin
+	DEPENDS:= +kea-libs +python3
+endef
+
+define Package/kea-ctrl
+	$(call Package/kea/Default)
+	TITLE+= Control
+	DEPENDS:= +kea-dhcp4 +IPV6:kea-dhcp6 \
+		+kea-dhcp-ddns
+endef
+
+define Package/kea-lfc
+	$(call Package/kea/Default)
+	TITLE+= lfc
+	DEPENDS:=+kea-libs
+endef
+
+define Package/kea-perfdhcp
+	$(call Package/kea/Default)
+	TITLE+= perfdhcp
+	DEPENDS:=+kea-libs
+endef
+
+CONFIGURE_ARGS += \
+	--with-log4cplus="$(STAGING_DIR)/usr" \
+	$(if $(CONFIG_PACKAGE_kea-perfdhcp),--enable-perfdhcp,)
+
+CONFIGURE_VARS += \
+	cross_compiling="yes"
+
+HOST_CONFIGURE_ARGS += \
+	--enable-static \
+	--enable-static-link \
+	--with-log4cplus="$(STAGING_DIR_HOSTPKG)" \
+
+HOST_CXXFLAGS += \
```

Comment:
Not sure about this. The TARGET_FLAGS make sense but not so much the HOST. I don't think space is a concern on the host.

Reply:
Well, removed it.